verticals/vertical-crm/errors.ts

```diff
@@ -0,0 +1,55 @@
+type ErrorOrigin = 'supaglue' | 'remote-provider'
+type ErrorParams = {
+  detail: string
+  status: number
+  code: string
+  origin?: ErrorOrigin
+  cause?: any
+}
+
+export class SGError extends Error {
```

Comment:
I would not use these error classes but rather trpc errors. However that can wait.